### DISCUSSION TITLE
Webhook Support

### DISCUSSION THREAD

**Discussion by kub3let** [shlinkio/shlink#2200] [Help wanted] [answered]
### Summary

I searched past issues and it seems webhook support got removed in v4.0.0, what was the reason behind that ?  #1340 

It would be great if one could define a webhook url, that get's called with a payload of the site metadata (short url, full url, slug, etc.)  and visitor details (remote ip, user agent, etc.)

### Use case

This way shlink could easily be integrated into other services, personally I want to trigger some events if a link is opened via shlink.

**Comment by acelaya** [comment, marked as answer]
The reason web hooks are no longer supported is explained in the issue you linked.

I'd recommend using any of the alternative supported integrations for real-time updates: https://shlink.io/documentation/advanced/real-time-updates/

**Reply by kub3let** [reply]
Thanks @acelaya, I'm using RabbitMQ now, works  great.
